frontend/src/components/WelcomePage.tsx

```diff
@@ -1,10 +1,19 @@
 import { Box, Flex, ListItem, Text, UnorderedList } from "@chakra-ui/react";
+import { useAsync } from "react-async-hook";
+import { Link } from "react-router-dom";
+import { getAllSongbooks } from "../services/songs";
+
+//
```

Comment:
Delete

Reply:
Assuming you meant the comment lines, deleted.